docs/components/query.md

```diff
@@ -115,7 +115,7 @@ Let's walk through all of those extensions:
 
 ### Partial Response
 
-QueryAPI and StoreAPI has additional behaviour controlled via query parameter called [PartialResponseStrategy](../../pkg/store/storepb/rpc.pb.go).
+QueryAPI and StoreAPI has additional behaviour controlled via query parameter called [PartialResponseStrategy](https://github.com/thanos-io/thanos/blob/main/pkg/store/storepb/rpc.pb.go).
```

Comment:
Thanks for the PR! 💫

These links are automatically checked and corrected by [`mdox`](https://github.com/bwplotka/mdox), which is our documentation automation tool, but currently isn't configured to correct such non-md relative links during website generation. 

A fix is already implemented in #4821 (you can check the [deploy preview](https://deploy-preview-4821--thanos-io.netlify.app/tip/components/query.md/) to verify)! 🙂

Also, the reason why docs check fails here is due to removing whitespace within the code blocks, as those are also autogenerated from CLI output by `mdox`.